clang/lib/Driver/ToolChains/CommonArgs.cpp

```diff
@@ -3125,6 +3125,8 @@ void tools::addMCModel(const Driver &D, const llvm::opt::ArgList &Args,
       else if (CM == "medany")
         CM = "large";
       Ok = CM == "small" || CM == "medium" || CM == "large";
+    } else if (Triple.getArch() == llvm::Triple::lanai) {
```

Comment:
I was wondering if there is an ordering here, but I can't really see one, so at end SG,